examples/text_to_speech_synthesizer/SpeechT5/README.md

```diff
@@ -0,0 +1,48 @@
+# Text to Speech synthesis with SpeechT5
+
+This is an example showing text to speech synthesis using SpeechT5 model. This has been verified to work on (linux-aarch64) Graviton 3 instance
```

Comment:
Thanks for adding the Speech Synthesis example for Graviton. It will be good to also see if support for the new WavLLM in https://github.com/microsoft/SpeechT5/tree/main/WavLLM added to the Microsoft SpeechT5 can also be included.